microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md

```diff
@@ -0,0 +1,95 @@
+
+The current version of the Microsoft Edge DevTools extension for Visual Studio Code fixes several issues that were reported by the community:
+
+*  The list of emulated devices in the screencast is now the same as the list of emulated devices in the browser.
+
+*  Device emulation now adds the correct user agent string, to trigger the correct display.
+
+*  You now get a warning when you use a deprecated launch version of Microsoft Edge, indicating that you'd miss out on features by using a deprecated version of Microsoft Edge.
+
+*  You can now provide arguments for the launched browser instance, in case you need to have a special setup:
+
+![Extension settings to specify launch arguments for Microsoft Edge.](../../media/2022/01/extension-settings-launch-arguments.png)
+
+
+<!-- ====================================================================== -->
+## Improvements and bug fixes for Focus Mode
+
+<!-- Title: Have you tried Focus Mode? -->
+<!-- Subtitle: To de-clutter and simplify the DevTools interface to focus on debugging web apps, enable Focus Mode in DevTools settings. -->
+
+Thanks to your early feedback, we've continued to make improvements and bug fixes on the new Focus Mode interface for Microsoft Edge DevTools.  Focus Mode is a new UI option that allows you to dock tools at the top or side of the DevTools window, and removes clutter from the toolbar.
+
+When using Focus Mode, all the same tools and [custom themes](../../../customize/theme.md) are available as in the existing DevTools UI.  Learn more about the [Focus Mode UI](../../../experimental-features/index.md#focus-mode), or try it for yourself by enabling the **Focus Mode** experiment in DevTools **Settings** > **Experiments**.
+
+![The Focus Mode UI.](../../media/2022/01/focus-mode.png)
+
+
+<!-- ====================================================================== -->
+## 3D View tool supports changing color themes in DevTools
+
+<!-- Title: 3D View better integrates with different themes in DevTools -->
+<!-- Subtitle: The 3D View tool now works when you select a different color theme in DevTools. -->
+
+In previous versions of Microsoft Edge, changing the theme in DevTools and then opening the 3D View tool resulted in a blank panel.  This issue has now been fixed, in Microsoft Edge 97.  To learn more, see [Apply color themes to DevTools](../../../customize/theme.md) and [3D View](../../../3d-view/index.md).
+
+![The 3D View tool now supports changing color themes.](../../media/2022/01/3d-view-with-color-theme.png)
+
+For more information about the issue that was fixed, see [Bug 36842109: 3D View panel blank after DevTools reload](https://dev.azure.com/microsoft/Edge/_workitems/edit/36842109/) and [Pull request 6615892: Fix for blank screen in 3D View after tools reloaded](https://dev.azure.com/microsoft/Edge/_git/chromium.devtools-frontend/pullrequest/6615892).
```

Comment:
```suggestion
```

These are ADO links and would not work for external readers. I maybe meant to leave them as a comment in the ADO work item if you needed extra info for the announcement. This entire paragraph should be removed.